<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_i_inventory_query_Dialog</class>
 <widget class="QDialog" name="w_i_inventory_query_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>199</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>frame_back_round</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>商品类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_ComboBox" name="comboBox_series">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>商品编码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_selectgoods" name="selectgoods" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>查询时间</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_querytime">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="lds_CheckBox" name="checkBox">
        <property name="text">
         <string>库存补仓提醒报表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar" name="widget"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="lds_tableView" name="tableView_sum"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="lds_tableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_CheckBox</class>
   <extends>QCheckBox</extends>
   <header>lds_checkbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_selectgoods</class>
   <extends>QWidget</extends>
   <header>lds_selectgoods.h</header>
   <container>1</container>
   <slots>
    <signal>okstr1(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>comboBox_series</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
